words-in-reverse.py

```diff
@@ -0,0 +1,16 @@
+words = input("Write a sentence containing 10 words (be creative): ")
+length = 10
+newwords = words.split(" ")
```

Comment:
@ebosetalee The link I shared with you yesterday was to get you familiar with python naming conventions. It's good to start developing that culture of abiding to PEP-8 standards early as it helps unify ways of writing code and makes others understand your code more easily when they come across it.
We typically place an underscore between words in a variable, so this should be `new_words`.

I'll start requesting for changes from tomorrow if you don't follow it.